common/app/routes/Hikes/components/Hike.jsx

```diff
@@ -10,7 +10,7 @@ export default contain(
     actions: ['hikesActions']
   },
   React.createClass({
-    displayName: 'Hike',
+    displayName: 'Video',
 
```

Comment:
Sorry, I thought this was what it say:  a "displayName", not a component name. I'll fix it.